django-backend/fecfiler/scha_transactions/migrations/0025_alter_schatransaction_transaction_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-10-04 13:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("scha_transactions", "0024_schatransaction_contact"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="schatransaction",
+            name="transaction_id",
+            field=models.TextField(max_length=20, null=True),
+        ),
+    ]
```

Comment:
Looks perfectly fine, but it's worth noting that #113 (Transaction Identifiers) also adds a migration 0025

Reply:
Yeah that conflict will have to be resolved once one of these gets merged

Reply:
The migration conflict has been resolved